### PR TITLE
#262 - terms and conditions

### LINKED ISSUE #262: Add terms and condition page
As of now, we mention terms and condition under the form that user can accept.
Screenshot from 2024-06-18 13-30-24

It should either be created and made accessible similarly to the policy, or removed.

### PR DESCRIPTION
**Pull request by AleksandraKozubal**
Legal advisor created one privacy policy for us, so I updated it and removed all things connected with legal view - it won"t be here.

image
It should close #262

### CODE REVIEW

File: lang/en/content.php [hunk 1]
```diff
@@ -248,23 +248,73 @@
...
-        ],
-        "conjunction" => "and",
-        "link_2" => [
-            "address" => "https://marketingplatform.google.com/about/analytics/terms/us/",
-            "description" => "google.com/analytics/terms",
-        ],
-        "subtitle_3" => "Do you have questions?",
-        "content_4" => "If you want to learn more about our Privacy Policy, use the contact form or write to us at:",
-        "email" => "hello@blumilk.pl",
+        "title_3" => "Obowiązująca od 6 września 2024 r.",
```
Comment (EwelinaSkrzypacz) [inline lang/en/content.php:251]: Shouldn't be this text in English? :eyes:

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. Why some mails are styled like I can click on that for example "office@blumilk.pl" but others don't like "iod@blumilk.pl"?
2. Why in browser card title I have "rzetwarzane dane"? 

image

File: lang/en/content.php (hunk 1 above)
Comment (AleksandraKozubal) [reply lang/en/content.php]: translated

**Comment by AleksandraKozubal** [comment]
> 1. Why some mails are styled like I can click on that for example "[office@blumilk.pl](mailto:office@blumilk.pl)" but others don't like "[iod@blumilk.pl](mailto:iod@blumilk.pl)"?
> 
> 2. Why in browser card title I have "rzetwarzane dane"?
> [image](https://private-user-images.githubusercontent.com/56546832/367657277-0c55d56c-17cb-4826-8de1-50c25a8adec6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY0NzA4NzYsIm5iZiI6MTcyNjQ3MDU3NiwicGF0aCI6Ii81NjU0NjgzMi8zNjc2NTcyNzctMGM1NWQ1NmMtMTdjYi00ODI2LThkZTEtNTBjMjVhOGFkZWM2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE2VDA3MDkzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2OWI2Zjc3YjRiOTViODBkZjk0NmVkMzEzMjk5NmUxNWZhYjQ3N2VmMzA3YmYzZmNkMDUzZDVjMmUzNGE3YTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KvNR25S0ek7nNWPllCbt_a6wsL4-xjtdaLyu-AGkBng)
> 
> 3. This number "7." is smaller that title on mobile view.  Is it by purpose? Also can we increase a bit line height when we have bullet point list? On mobile, texts in bullet point list look a bit squezzed
>
> [image](https://private-user-images.githubusercontent.com/56546832/367658457-345421e8-b511-42e8-835f-3fc8ab5f8019.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY0NzA4NzYsIm5iZiI6MTcyNjQ3MDU3NiwicGF0aCI6Ii81NjU0NjgzMi8zNjc2NTg0NTctMzQ1NDIxZTgtYjUxMS00MmU4LTgzNWYtM2ZjOGFiNWY4MDE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTE2VDA3MDkzNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwMjA3YmFmYTIzZjdmNDk4MDNlNjE4OTQ5YjBkNjhkN2I3YWFmYzU2Yjg1NDc0NjQyMjhjOGRjMTU0MzhhODcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.c_mIkbaHR2Vb5Nlqx7b7Jg0Wzm4Ohme9173RXXAeX1g)

1. 2. fixed
3. Changed numbers size and added those spaces and line height.

**Review by EwelinaSkrzypacz — COMMENTED** [review]
Please link email here too

image

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**